### ISSUE TITLE
Ability to check address: Do I live in City of Seattle?

### ISSUE THREAD

**Issue by elisekalstad** [civiform/civiform#6812]
#### Is your feature request related to a problem? Please describe.

During a training with 15 CBO case navigators, we heard that it is helpful if there was a link to check if their client lives within the City of Seattle before applying to programs. 

#### Describe the solution you'd like.

Matt had a lightbulb moment--I'll let him fill this in. Perhaps a link to the Open Seattle GIS map that checks if the address is in Seattle: https://data-seattlecitygis.opendata.arcgis.com/

#### What priority is this?

Medium--it would be nice to have. We could also consider adding this to the Assistance & Discounts page.

#### Associated civic entity

Seattle and the CBOs working on Fresh Bucks

#### User Group

If this request or piece of feedback comes from a resident, add the label 'Resident feedback'.
If this request or piece of feedback comes from a CBO or TI, add the label 'TI'.
If this request or piece of feedback comes from an admin, add the label 'Program Feedback'.

#### Done when:

What conditions need to be met before this issue can be closed?

**Comment by msprenke** [comment]
perhaps this could be built into the discoverability feature.  Enter your address and only see the programs you qualify for

**Comment by elisekalstad** [comment]
Additional links and notes from Fresh Bucks: As we chatted about yesterday, I'm aware of these two resources to check if an address is in the City of Seattle. Not sure if there are others that are clearer and more accessible.
Find your council district: https://www.seattle.gov/council/meet-the-council/find-your-district-and-councilmembers
If nothing comes up, not in City of Seattle, which may be confusing since it's not a clear answer
GIS City Limits: https://data-seattlecitygis.opendata.arcgis.com/datasets/SeattleCityGIS::seattle-city-limits-2/explore
I think this is accessible to the public ?
Have to click magnifying glass in top right corner to enter address
I like how it clearly identifies the City boundaries

**Comment by gwendolyngoetz** [comment]
What if we enabled the `Common Intake` feature and create a program that asked for their address?

**Comment by msprenke** [comment]
Is common intake ready?  I dont think I've seen anything about it in a while

**Comment by dkatzz** [comment]
> I thought it was a done feature that we just didn't use. @nb1701 / @rockycodes, Common Intake is done right?

Yeah it's done! Just ensure `INTAKE_FORM_ENABLED` is set to enabled in the settings panel and you can see docs for how to set it up [here](https://docs.civiform.us/user-manual/civiform-admin-guide/working-with-programs/set-pre-screener).

**Comment by msprenke** [comment]
moving to backlog.  There are other ways to determine this such as the pre screener and most programs include eligibility up front.

**Comment by MrkAnthony** [comment]
Hello,
My team and I would like to work on implementing a solution for this feature. Could you please assign this issue to us?

Thank you!

**Comment by melanie-exygy** [comment]
Hi @MrkAnthony, thanks for your interest in contributing to CiviForm!  Here is some information on how best to start:  [Technical contribution guide](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#community-contributors-not-part-of-exygy-google-or-a-civic-entity).  Please introduce yourself on our [Slack channel](https://civiform.slack.com/archives/C07S691HDU0) and we'll help you find a good first issue to work on.

**Comment by msprenke** [comment]
Since the origination of this request comes from TI's, we should look at a solution that addresses their need.  Residents could accomplish this with a prescreener.  TI's may benefit from adding client address to client set ups, this would instantly tell TI's if the client doesnt qualify for in city services when they move to the program cards page after selecting the client.
